.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,46 @@
+# Contribution guidelines
```

Comment:
Can you remove this file please. It's not that we shouldn't have it, it's just not correct.